src/main/Recursion.java

```diff
@@ -0,0 +1,39 @@
+package main;
+
+public class Recursion {
+    public static void main(String[] args) {
+        System.out.println(fizzBuzz(15));
+        polindrome("this is me".replace(" ", ""));
+
+    }
+    static String fizzBuzz(int num){
+        if (num%3==0 && num%5==0){
+            return "Fizz_Buzz";
+        }
+        else if (num%5==0){
+            return "Fizz";
+        }
+        else if (num%3==0){
+            return  "Buzz";
+        }
+        else{
+            return "1";
+        }
```

Comment:
There is no recursion (Recursion is a process of a method calling itself.), please check this article https://www.geeksforgeeks.org/recursion-in-java/ and then fix it.